### PR TITLE
Fix filtering cannot use relational reflection fields

### PR DESCRIPTION
**Pull request by star2000**
Closed https://github.com/strawberry-graphql/strawberry-graphql-django/issues/84

### CODE REVIEW

File: strawberry_django/filters.py [hunk 1]
```diff
@@ -72,6 +74,19 @@ def filter_deprecated(model, *, name=None, lookups=False):
...
+def get_field_names_from_opts(opts):
+    '''
+    copy from django.db.models.sql.query.get_field_names_from_opts
+    because get_field_names_from_opts not in django.db.models.sql.query.__all__
+    and no cache optimization
+    '''
+    return set(chain.from_iterable(
+        (f.name, f.attname) if f.concrete else (f.name,)
+        for f in opts.get_fields()
+    ))
```
Comment (bellini666) [inline strawberry_django/filters.py:87]: It's not a problem that the function is not in `__all__`. That only matters when you do `from <some_place> import *`

Just import it from there and use it directly :)

**Review by bellini666 — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution and sorry for the late review. Just a small change and this should be good to merge

File: strawberry_django/filters.py (hunk 1 above)
Comment (star2000) [reply strawberry_django/filters.py:87]: I think functions not in __all__ are meant for internal use only and may be renamed or removed later

**Review by bellini666 — APPROVED** [review]

**Comment by bellini666** [comment]
Thanks for the contribution :)

**Pull request merged**